boxoffice/templates/boxoffice.html

```diff
@@ -69,7 +69,14 @@ <h1 class="category-heading">{{title}}</h1>
                           {{/}}
                         </div>
                         {{#if line_item.quantity_available > 0}}
-                          <p class="valid-upto"><i class="fa fa-fw fa-long-arrow-up"></i>Price goes up after {{line_item.price_valid_upto}}</p>
+                          <p class="valid-upto">
+                            {{#if line_item.has_higher_price}}
+                              <i class="fa fa-fw fa-long-arrow-up"></i>
+                              Price goes up after {{line_item.price_valid_upto}}
```

Comment:
Isn't that still "goes up on"? We shouldn't be picking 11:59 PM as end-point. It should be a non-inclusive endpoint at midnight.